src/GraphQL/Introspection/__Type.cs

```diff
@@ -52,7 +52,7 @@ public __Type()
                 }),
                 context =>
                 {
-                    if (context.Source is ObjectGraphType || context.Source is InterfaceGraphType)
+                    if (context.Source is ObjectGraphType || context.Source is InterfaceGraphType || context.Source is InputObjectGraphType)
```

Comment:
Looking at the reference implementation again, this is correct behavior to exclude the `InputObjectGraphType` here.   There is `inputFields` where the input object data is queried from.

https://github.com/graphql/graphql-js/blob/master/src/type/introspection.js#L287-L295